jit/v2/JIT_v2.0.0.yaml

```diff
@@ -730,20 +730,20 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
-  '/v2/transport-calls/{transportCallID}/omit':
+  '/v2/te-minalcalls/{terminalCallID}/omit':
```

Comment:
typo
/v2/terminalcalls/

Reply:
fixed to `/v2/terminal-calls/{terminalCallID}/omit`